### PR TITLE
upload_apk script updated

### PR DESCRIPTION
**Pull request by neel1998**
Fixes upload script updated

**Changes**: command to delete and recreate apk branch added

### CODE REVIEW

**Comment by neel1998** [comment]
yes yes. I am on it

**Comment by adityastic** [comment]
Distribution issue. Travis shifted to Xenial, Get it back on trusty

**Comment by neel1998** [comment]
@CloudyPadmal  done.

File: scripts/upload-apk.sh [hunk 1]
```diff
@@ -51,15 +47,18 @@ if [ "$TRAVIS_BRANCH" == "$DEVELOPMENT_BRANCH" ]; then
...
-    git checkout apk
-    git add -A
-    git commit -am "Travis build pushed [development]"
-    git push origin apk --force --quiet> /dev/null
 fi
+git checkout apk
+git add -A
+git commit -am "Travis build pushed [$TRAVIS_BRANCH]"
+
+git branch -D apk
```
Comment (CloudyPadmal) [inline scripts/upload-apk.sh]: Cannot delete as it's already checked into apk branch

Comment (adityastic) [reply scripts/upload-apk.sh]: Yeah, @neel1998 switch to temporary branch. I should have noticed it while reviewing, my bad

**Pull request merged**